src/main/java/com/wwi21sebgroup5/cinema/components/Setup.java

```diff
@@ -0,0 +1,41 @@
+package com.wwi21sebgroup5.cinema.components;
+
+import com.wwi21sebgroup5.cinema.entities.City;
+import com.wwi21sebgroup5.cinema.services.CityService;
+import jakarta.annotation.PostConstruct;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+@Component
+public class Setup {
+
+    @Autowired
+    CityService cityService;
+
+    @Autowired
+    private CsvDataLoader csvDataLoader;
+
+    private static final String CITY_FILE = "static/data/cities.csv";
+
+    /**
+     * Component which runs after start-up and initializes static data
+     */
+    @PostConstruct
+    private void setupData() {
+        setupCities();
+    }
+
+    /**
+     * Setup workflow for cities stored in a csv-file
+     */
+    private void setupCities() {
+        List<City> cities = csvDataLoader.loadObjectList(City.class, CITY_FILE);
+
+        for (City city : cities) {
+            cityService.save(city);
```

Comment:
Wie vorher gesagt, das dürfte schlecht für die Performance sein

Reply:
Dauert beim Start-Up ca. 12 Minuten länger, würde das in ein Sunmodul auslagern und dann je immer nur ein mal bauen lokal, ändert sich ja nicht. In der CD-Pipeline sind 10 Minuten ja egal.

Garantiert halt keine Hirnbrei Städte